packages/locale/en.yml

```diff
@@ -24,27 +24,23 @@ why-it-works:
     the learning process is shaped according to their needs.
   teamwork.title: Teamwork
   teamwork.description: >
-    Participate in personalized individual mentoring and group projects. It is
-    you who chooses how to contribute to the community.
+    Participate in trail mentorships and group projects. You choose how to contribute to the community.
 
 mentoring:
   title: Our methodology
   description: >
     It is structured by mentorships that are guided by paths according
     to the needs and time of each person, focusing on access to the job market.
-  personalized-mentoring.title: Personalized mentoring
-  personalized-mentoring.description: >
-    Personalized mentoring aims at individual monitoring of the mentee, focused
-    on introductory trails and specialized exercises.
-  pair-mentoring.title: Pair mentoring
-  pair-mentoring.description: >
-    Pair mentoring aims to exchange experience between more than one mentee, pair
-    programming and collaboration in the development of exercises and projects.
+  self-guided-mentorships.title: Self-guided mentorships
+  trail-mentoring.description: >
+    Self-guided mentorships aims to guide the mentee in their beginning studies with introductory self-guided mentorships and specialized exercises.
```

Comment:
```suggestion
    Self-guided mentorships aims to guide the mentee in their beginning studies with introductory subjects and  exercises.
```

Reply:
Por favor considere minha nova sugestão @Guilherme-BS 

Title: Study mentorship
Text: "Our study mentorship aims to help new learners at the beginning of their journey as a software developer. By following predefined knowledge paths with the help of a mentor 

O comment anterior ficou outdated :p